backend/game.js

```diff
@@ -1,16 +1,16 @@
 const crypto = require("crypto");
 const path = require("path");
-const {shuffle, truncate} = require("lodash");
+const { shuffle } = require("lodash");
 const uuid = require("uuid");
 const jsonfile = require("jsonfile");
 const Bot = require("./player/bot");
 const Human = require("./player/human");
-const Pool = require("./pool");
+const PoolBuilder = require("./pool/poolBuilder");
```

Comment:
my gut feeling is this should require `pool` and that `poolGenerator` or whatever it's called should be required inside pool so you can just go `pool.create` or `pool.generate` and then that calls out to the booster and pool creating functions.

Reply:
I think we have a pretty similar idea of the target design here, but that we are being caught up on naming a little.

I think that PoolBuilder would be more appropriately called 'Pool', but I chose PoolBuilder because I didn't want to create confusion and conflicts with the existing usage of 'pool' in this project (where pool is an array of cards, as in Game.pool).

In future patches I see removing the current usage of 'pool' completely:
- the pool/pool component would be integrated into various implementations of PoolBuilder that would be injected into Game externally
- the current use of 'pool' in Game.js (this.pool) could be more appropriately called (this.packs) or some such
  - (Which would signal that this.packs is a concrete representation of the exact cards to be used, instead of the more abstract idea of a Pool as the rules that define the bounds of the possible cards to be used that PoolBuilder is currently capturing.)

At that point I think we could (and should) rename PoolBuilder to Pool in a way that matches your intuition here.